src/js/ui/simply-pebble.js

```diff
@@ -140,7 +207,43 @@ var colorMap = {
 };
 
 var Color = function(color) {
-  return colorMap[color] ? colorMap[color] : colorMap.clear;
+  if (color.charAt(0)=='#') {
+      //Convert full hex to shorthand
```

Comment:
Spaces between binary operators (such as `==`). Always use strict equality `===`. Space after `//` for the two comments here.